### DISCUSSION TITLE
Problema password dopo aggiornamento firmware tasmota

### DISCUSSION THREAD

**Discussion by MauroGiudici** [indomus/forum#320] [Hardware & Firmware]
Buongiorno a tutti. Da qualche giorno continuo a sbattere la testa per alcuni problemi dei miei sonoff 4ch pro tasmotizzati alla versione 6.5. Alexa non li rileva più come hue. Ho provato allora ad aggiornare il firmware a versioni più recenti ma ogni volta che ci provo (sia con tasmotize che con Visual studio code) arrivo sempre allo stesso punto. Il firmware viene scritto, il dispositivo hardware viene visto dalla rete del pc sotto l'indirizzo http://192.168.4.1/, quando mi connetto mi esce da inserire la rete wifi e la password ed ogni volta che la inserisco torna alla stessa pagina con la scritta "Connect failed to Joker Please, check your credentials". Ogni volta cancello il firmware con esptool.py e rifaccio tutto da capo. con qualsiasi versione mi da lo stesso problema. cosa può essere?

**Comment by Scialla** [comment]
Immagino che Joker sia il nome della tua WiFi... Se non riesce a connettersi sei sicuro di avere inserito correttamente nome della WiFi e password? Sei sicuro che la rete sia a 2,4 GHz? Io proverei pure a riavviare il modem/router

**Reply by MauroGiudici** [reply]
Si è il nome della Wifi. Il router è settato su 2.4 GHz e la password è sempre la stessa (in wep). La cosa strana è che se utilizzo firmware sino al 6.6 la rete viene vista, sui successivi non mi accetta le credenziali. ho provato anche tasmotizer con login e password ma è sempre la stessa cosa.

**Reply by Scialla** [reply]
magari una incompatibilità con WEP? (è obsoleta) Prova a impostarla in WPA

**Reply by MauroGiudici** [reply]
Ho provato WPA2 e finalmente ha accettato la password. con le versioni più recenti del firmware WEP non viene accettata.

**Reply by snakuzzo** [reply]
...e mi sembra pure giusto che non sia più accettata ;)
Bene così

**Comment by umibozu83** [comment]
provato con tastmotize , ti da anche la possibilità di settare la rete wifi

**Reply by MauroGiudici** [reply]
Si,  ho provato anche tasmotizer con login e password ma è sempre la stessa cosa. Ho anche provato a settare nel router password diverse ma il risultato è sempre quello. è come se il router non mi "accettasse" quel device.
